### PR TITLE
#75: add English and Polish localizaton

### PR DESCRIPTION
**Pull request by pawelFelcyn**
This pull request refers to https://dev.azure.com/survey-project/Survey/_boards/board/t/Survey%20Team/Issues/?workitem=75 adds the following to the admin panel:
- English and Polish localization
- by default, it loads the browser language (if it is not English or Polish, it loads English)
- on the right side of the top bar, there is a picker that allows the user to change the selected language

### CODE REVIEW

File: src/assets/i18n/pl.json [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+    },
+    "respondents":{
+        "addRespondents":{
+            "quantity": "Ilość",
+            "validationError": "Wartość musi być między 1 a 400",
+            "create": "Utwórz",
+            "copyToClipboard": "Skopiuj do schowka"
+        },
+        "respondents": {
+            "createRespondentsAccounts": "Utwórz konta dla respondentów",
```
Comment (mikolaj4) [inline src/assets/i18n/pl.json:25]: It does not work while changing language to polish.

File: src/presentation/modules/app/components/add-respondents/add-respondents.component.html [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
   <form (ngSubmit)="submit()" class="form-container" [formGroup]="form!">
     <mat-form-field appearance="outline" class="amount-input">
-        <mat-label>Ilość</mat-label>
+        <mat-label> {{ 'respondents.addRespondents.quantity' | translate }} </mat-label>
         <input matInput type="number" formControlName="amount" [(ngModel)]="dto.amount">
-        <mat-error *ngIf="form?.invalid">Wartość musi być między 1 a 400</mat-error>
+        <mat-error *ngIf="form?.invalid">{{ 'respondents.addRespondents.validationError' | translate }}</mat-error>
     </mat-form-field>
-    <button mat-flat-button color="primary">Utwórz</button>
+    <button mat-flat-button color="primary">{{ 'respondents.addRespondents.create' | translate }}z</button>
```
Comment (mikolaj4) [inline src/presentation/modules/app/components/add-respondents/add-respondents.component.html:8]: Typo with "z"

**Review by mikolaj4 — CHANGES_REQUESTED** [review]
2 minor changes

File: src/assets/i18n/pl.json (hunk 1 above)
Comment (pawelFelcyn) [reply src/assets/i18n/pl.json:25]: it's fixed

**Pull request merged**